crates/swift-bridge-ir/src/bridged_type/bridged_opaque_type.rs

```diff
@@ -434,7 +444,7 @@ impl BridgeableType for OpaqueForeignType {
                     format!("{{ if let val = {expression} {{ val.isOwned = false; return val.ptr }} else {{ return nil }} }}()", expression = expression,)
                 }
                 HostLang::Swift => {
-                    format!("{{ if let val = {expression} {{ return Unmanaged.passRetained(val).retain().toOpaque() }} else {{ return nil }} }}()")
```

Comment:
`passRetained` already increments the retain count. We do not want to increment it twice.